_source/logzio_collections/_log-sources/opentelemetry.md

```diff
@@ -1,48 +1,57 @@
 ---
-title: Ship traces with OpenTelemetry to Logz.io
+title: Ship traces with OpenTelemetry
 logo:
-  logofile: cisco-meraki-logo.png
-  orientation: horizontal
-data-source: Jaeger, Zipkin, OpenCensus, OpenTelemetry
+  logofile: opentelemetry-icon-color.png
+  orientation: vertical
+data-source: OpenTelemetry
 description: How to deploy an OpenTelemetry Collector for traces to logz.io
 open-source:
   - title: Logz.io-OpenTelemetry trace exporter
     github-repo: https://github.com/open-telemetry/opentelemetry-collector-contrib
 contributors:
   - yyyogev
+  - yberlinger
 shipping-tags:
   - traces
 ---
 
-Logz.io's trace exporter for OpenTelemetry allows you to ship distributed traces to Logz.io from different APM agents, such as Jaeger, Zipkin, and so on.
-
-
-#### How it works
+Logz.io's trace exporter for OpenTelemetry allows you to ship distributed traces to Logz.io from different APM (Application Performance Management/Monitoring) agents, such as Jaeger, Zipkin, and so on.
 
 The OpenTelemetry Collector pipeline has the following main components: 
+
 * Receivers
 * Processors 
-* Exporters. 
+* Exporters
 
-OpenTelemetry also includes extentions for additional functionality such as diagnostics and health checks. OpenTelemetry has a dedicated collector for contributed components, such as the Logz.io exporter.
+OpenTelemetry also includes extentions for additional functionality such as diagnostics and health checks, as well as a dedicated collector for contributed components, such as the Logz.io exporter.
```

Comment:
@yyyogev - What are "contributed components?

Reply:
Any components (receivers, processors, exporters, and extensions) in the collector that external source made, like us.

Reply:
@yyyogev  - could I call them 3rd party components

Reply:
I guess..